internal/cli/root/atlas/builder.go

```diff
@@ -116,7 +117,6 @@ func Builder() *cobra.Command {
 		Use:     atlas,
 		Short:   "CLI tool to manage MongoDB Atlas.",
 		Long: `The Atlas CLI is a command line interface built specifically for MongoDB Atlas. You can manage your Atlas database deployments and Atlas Search from the terminal with short, intuitive commands.
-		
```

Comment:
can you please rollback this change

Reply:
Addressed in [3ec5c80](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/3ec5c80d1c90d9349ea67e1ead00e041f7c3b1bf)

Reply:
Resolved in [ad506a4](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/ad506a4da8ee721d4727dbe9259eabd8bce5899b)